invenio_records_rest/utils.py

```diff
@@ -139,7 +166,7 @@ def data(self):
                 response = make_response(jsonify(data), data['status'])
                 response.headers['Location'] = location
                 abort(response)
-            except BuildError:
+            except (BuildError, ValueError):
```

Comment:
OK, In the last implementation the whole mapping is built at module initialization (or anyway first call of the `cached_property`, so all inconsistencies should be good.